webapp/utils/__init__.py

```diff
@@ -1,6 +1,43 @@
+import base64
+import json
+from typing import Any
+
+import jwt
+import requests
+
 from webapp.exceptions import ECSTaskDoesNotExistError
 from webapp.utils.aws import CloudWatchLogsClient, ECSClient
 
+REGION = "us-east-1"
```

Comment:
I would propose we set this in the app `Config` class, possibly with a default of `us-east-1`, but could be overridden with an env var.

We could then set this constant in this utility function from that, or even call it directly where used.

Reply:
[The environment variable is added to the`webapp.Config` as an `OPTIONAL_ENV_VAR`](https://github.com/MITLibraries/alma-sapinvoices-ui/pull/60/commits/40227568eb85d1c3cd24e2b5b8be5e89d590fd27#diff-966b49fcc05ab61f6b2ef9d13713ceb9da0b5d825b9ef5cdb16961b931e6bfd6R69-R73); the environment variable is set by AWS by default (i.e., region name wherever app is deployed), and [`webapp.utils.parse_oidc_data` has been updated to use `Config().AWS_DEFAULT_REGION`](https://github.com/MITLibraries/alma-sapinvoices-ui/pull/60/commits/40227568eb85d1c3cd24e2b5b8be5e89d590fd27#diff-20ff21c90d95c84f47465f8287c3e0b6524b04e0fffad8e50a452f9f32e52cdfR28-R33).